packages/jwt-verifier/lib.js

```diff
@@ -51,10 +51,28 @@ class OktaJwtVerifier {
         delete jwt.body;
         const errors = [];
         for (let claim of Object.keys(this.claimsToAssert)) {
-          const actualValue = jwt.claims[claim];
           const expectedValue = this.claimsToAssert[claim];
-          if (actualValue !== expectedValue) {
+          let op = undefined;
```

Comment:
Not to unduly criticize unrequested bounties of free code upgrades, but I'd love to see this new code pulled out into some smaller separate functions.